package-overrides/npm/postcss-import@8.0.2.json

```diff
@@ -0,0 +1,10 @@
+"map": {
+  "object-assign": "npm:object-assign@4.0.1",
```

Comment:
Maps shouldn't include exact versions because version dependency ranges are used in the package dependencies.